### PR TITLE
recipe for pylibseq 0.1.9

### PR DESCRIPTION
**Pull request by molpopgen**
* [X] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by molpopgen** [comment]
@bioconda/core Ready for review.  Routine version number update.

File: recipes/pylibseq/0.1.8/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+source:
+  fn: pylibseq-0.1.8.tar.gz 
+  url: https://pypi.python.org/packages/61/a3/2777266d22ace0f461bb323d6ad2a47d67bcaff68b203ff1b0637adc77da/pylibseq-0.1.8.tar.gz
+  md5: 53ceb50b9c454a2a23890319d06fd57f
+
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
+  string: "py{{CONDA_PY}}_{{PKG_BUILDNUM}}"
```
Comment (bgruening) [inline recipes/pylibseq/0.1.8/meta.yaml]: You could remove this string.

**Comment by molpopgen** [comment]
ok.  changes made.  merge?

**Comment by bgruening** [comment]
Thanks! :)

**Pull request merged**